test/cmd/status.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc delete project project-bar
+  exit 0
+) &>/dev/null
+
+login_kubeconfig="${ARTIFACT_DIR}/login.kubeconfig"
+cp "${KUBECONFIG}" "${login_kubeconfig}"
+
+os::test::junit::declare_suite_start "cmd/status"
+# login and ensure no current projects exist
+os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
+os::cmd::expect_success 'oc delete project --all'
+os::cmd::try_until_text "oc get projects -o jsonpath='{.items}' 2>/dev/null" "^\[\]$"
```

Comment:
This is to prevent the race condition that https://github.com/openshift/origin/issues/12667 to happen (some tests below would fail if `oc get projects` expected no projects, but some were still deleting)
@smarterclayton fyi